invokeai/app/cli_app.py

```diff
@@ -145,7 +141,7 @@ def invoke_cli():
     db_location = os.path.join(output_folder, "invokeai.db")
 
     services = InvocationServices(
-        generate=generate,
+        model_manager=model_manager,
```

Comment:
Why does the API version register generator factory but this one registers model manager?

Reply:
The API version was a mistake inherited from the previous version which returned `InvokeAIGeneratorFactory` objects, which neither you nor @keturn thought necessary. So now there is a ModelManager service used to retrieve the models passed in turn to `Txt2Img()`.